src/collection.coffee

```diff
@@ -43,13 +46,20 @@ class window.Brainstem.Collection extends Backbone.Collection
   lastFetchOptions: null
   firstFetchOptions: null
 
+  model: (attrs, options) ->
+    Model = require('./model')
```

Comment:
This `require` has to be below `module.exports` here due to circular dependencies.